test/metabase/models/permissions_test.clj

```diff
@@ -706,7 +706,9 @@
 (deftest grant-revoke-root-collection-permissions-test
   (mt/with-temp PermissionsGroup [{group-id :id}]
     (letfn [(perms []
-              (db/select-field :object Permissions :group_id group-id))]
+              (db/select-field :object Permissions {:where [:and
+                                                            [:<> :object (perms/general-perms-path :subscription)]
```

Comment:
Same thing I mentioned about the other test, instead of hardcoding a specific `remove`-style clause like `<> '/general/subscription` it's better to future-proof this and just do a `filter`-style clause that keeps the rows we do want so we don't have to go do this again next time we add a general perm we want to grant everyone by default